Library/Formula/cdiff.rb

```diff
@@ -0,0 +1,15 @@
+class Cdiff < Formula
+  desc "View colored, incremental diff in workspace or from stdin with side by side and auto pager support"
+  homepage "https://github.com/ymattw/cdiff"
+  head "https://github.com/ymattw/cdiff.git"
+  url "https://github.com/ymattw/cdiff/archive/0.9.7.tar.gz"
+  sha256 "41eff442efc34e545d89b94f47da22ffc5edb4224476728746841b64b69f3512"
+
+  def install
+    system "python", "./setup.py", "install", "--prefix", prefix
+  end
+
+  test do
+    system "cdiff --version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!